server/controllers/stock/core.js

```diff
@@ -279,6 +279,7 @@ function getLotsMovements(depotUuid, params) {
     JOIN flux f ON f.id = m.flux_id
     LEFT JOIN document_map dm ON dm.uuid = m.document_uuid
     LEFT JOIN service AS serv ON serv.uuid = m.entity_uuid
+    LEFT JOIN patient AS pa ON pa.uuid = m.entity_uuid
```

Comment:
Can you instead use the `entity_uuid` to filter on patient?  That would save a join.  I'm pretty sure the `entity_uuid` here is actually the `patient.uuid`.

Reply:
Okay